### PR TITLE
Fixing errors as skill

### PR DESCRIPTION
**Pull request by AndrzejRomaniuk**
Pull request related to issue #1394 

This is my first pull request, I hope I am doing everything right. 
(I also added my name  in the author's list, but am not certain I should do that...)

### CODE REVIEW

**Review by dpshelio — DISMISSED** [review]
thanks @AndrzejRomaniuk - I've not read it yet your contrubition in detail yet, but a quick comment. Could you break the two paragraphas (long lines) into multiple shorter lines? Markdown would still render them properly as two bullet points, but it would be easy to track future changes with version control.

**Review by AndrzejRomaniuk — COMMENTED** [review]
Changes done for the request (breaking one text line in raw into multiple lines)

File: AUTHORS
Comment (zkamvar) [reply AUTHORS:61]: Given https://carpentries.org/blog/2022/07/thank-you-francois/, I can help field this one. In short, No, we do not use this file for authorship anymore. And for what it's worth, much like authorship in academia in general, the model we use is useful, but needs improvement. 

2015-2017: we used the AUTHORS file via https://github.com/swcarpentry/swc-releases#readme

2018-present: We matched the git history with AMY records using the [`generate_zenodo_json()` function](https://github.com/fmichonneau/chisel/blob/7b3fbf41e3f46e2450c726c94be6540c6b6e16a9/R/release.R#L412-L462) from François' R package chisel. @ErinBecker and I will be taking charge of this process for the lesson releases that will happen prior to the switch to The Workbench. 

Future: @MCMaurer and @annajiat [launched a lesson publication task force in 2021](https://carpentries.org/blog/2021/10/lesson-publication-task-force/) to tackle the notion of authorship credit in lessons that should smooth out this process moving forward.

Comment (brownsarahm) [reply AUTHORS:61]: Thanks! we'll leave this edit, but @AndrzejRomaniuk to ensure credit in the fort term, make sure to update your AMY profile to indicate that you want credit for lesson contributions.

Comment (AndrzejRomaniuk) [reply AUTHORS:61]: Thank you very much @zkamvar and @brownsarahm for the comments :) 

@brownsarahm, if possible to ask, where in AMY can I do that? I understand such information would go to "Your workshop activity" or similar section, but as far as I can tell I cannot edit it directly, apart from filling a self-organised workshop form.

Comment (zkamvar) [reply AUTHORS:61]: @AndrzejRomaniuk, if you go to edit your profile, and select the "consents" tab, the first question is about giving consent to lesson publication.

screenshot of "consents" tab in AMY for Zhian Namir Kamvar's user profile. Five tabs are visible: Person, Awards, Tasks, and Community Roles. The visible question asks "Do you consent to have your name or identity associated with lesson publications?". The selected option from a dropdown says "Yes, and use the name associated with my profile."

Comment (AndrzejRomaniuk) [reply AUTHORS:61]: Dear @zkamvar  I don't have such tab arrangement active on my AMY account; only viewing, updating and training tabs shown. However, in an update tab I have the same consent option, changed to what you have shown in an example

Comment (zkamvar) [reply AUTHORS:61]: What you did is correct. For context: what I showed was the admin view for AMY... I had momentarily forgotten that it would not be the same view.

Comment (brownsarahm) [reply AUTHORS:61]: thanks @zkamvar

File: _episodes/17-live.md
Comment (karenword) [inline _episodes/17-live.md:96]: Maintainers have discussed this contribution and agreed it should be shortened before inclusion, with lengthier content reserved for future reference.
```suggestion
+ **Debugging as a skill** – while learners may have negative associations with "mistakes" or "typos", it may be useful to re-frame troubleshooting as the routine process of finding and fixing "bugs". 
+ Learning to parse error messages is an important step in developing debugging skills.
```

File: _episodes/17-live.md
Comment (karenword) [inline _episodes/17-live.md:104]:
```suggestion
+ **Intentional vs accidental errors** - It can be useful to intentionally demonstrate common mistakes and error messages. Alternatively, mistakes made by the Instructor or learners can offer useful opportunities to learn about and positively frame errors.
```

**Comment by karenword** [comment]
Suggested changes per conversation with @brownsarahm and @ndporter

**Comment by AndrzejRomaniuk** [comment]
> Suggested changes per conversation with @brownsarahm and @ndporter

Saw the changes and agree it may be better to shorten the contribution, and save a more complex explanation for later reference. However, if so, after the merge it may be good to leave two related issue threads open as both can provide additonal ideas for the future use?

**Pull request merged**
